docs/advanced-features/compiler.md

```diff
@@ -252,7 +251,7 @@ module.exports = {
 }
 ```
 
-If you have feedback about `swcMinify`, please share it on the [feedback discussion](https://github.com/vercel/next.js/discussions/30237).
```

Comment:
Let's leave this feedback link for now so it's still going to a consistent place